docker/core/http/docker-compose.yml

```diff
@@ -1,15 +1,16 @@
 version: "3.5"
 services:     
     designer:
-        image: fogflow/designer:3.0
+        image: fogflow/designer:latest
```

Comment:
Better to write valid number here. like 3.1 instead of latest  @showersky  could you please suggest